Arrowgene.Ddon.Database/DdonDatabaseBuilder.cs

```diff
@@ -24,6 +29,16 @@ public static IDatabase Build(DatabaseSetting settings)
                 _ => throw new ArgumentOutOfRangeException($"Unknown database type '{settings.Type}' encountered!")
             };
 
+            database.Migrator = new DatabaseMigrator(new List<IMigrationStrategy>()
+            {
+                new MailMigration(settings)
```

Comment:
We should use reflection like the other migration frameworks and then get all the migration files from the assembly.

Reply:
All other systems in our codebase (think DdonGameServer and EntitySerializer) use this approach of stuffing the instances in a big list. For consistency I've done the same here.

Reply:
Just because the other two systems are awful, that should not be a reason to make this one awful as well :). I would actually suggest we do the opposite where those ones are also learned via reflection APIs. There is too much busy work to make a simple packet or CData type.